tests/cross_functional/kcs/test_monitor_recovery.py

```
@@ -308,6 +336,37 @@ def patch_sleep_on_osds(self):
+            pod_type (str): Type of pod ("osd" or "mon"). Defaults to "osd".
+
+        Raises:
+            ValueError: If the pod_type is neither "osd" nor "mon".
+
+        """
+        if pod_type == "osd":
+            pod_objs = get_osd_pods()
+        elif pod_type == "mon":
+            pod_objs = get_mon_pods()
+        else:
+            raise ValueError(f"Invalid pod type: {pod_type}. Use 'osd' or 'mon' ")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)